docs/platform_operators/external-databases.md

```diff
@@ -142,10 +205,15 @@ In the following section, an external database is created using the bitnami post
 
 1. [Install cf-for-k8s](../deploy.md)
 
-    i. Render the final K8s template to raw K8s configuration. Pass the `db-values.yaml` file as additional file to `ytt`
+    i. Configure your [`cf-values.yml`](../deploy.md#cf-values) file
+    i. Render the final K8s template to raw K8s configuration. Pass the `db-values.yml` file as additional file to `ytt`
```

Comment:
> Render the final K8s template to raw K8s configuration

Could you please rephrase this? Maybe instead of "raw K8s configuration" I'm more used to hearing "raw K8s yaml" ?

Reply:
OK